src/tensorflow_utils.cc

```diff
@@ -25,9 +25,8 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "tensorflow_utils.h"
-
```

Comment:
Keep this empty line. Convention is leave an empty line after the header for the file. And the line after 
```
#include "triton/backend/backend_common.h"
```
As well